<ui version="4.0" >
 <class>ManageCaptures_UI</class>
 <widget class="QDialog" name="ManageCaptures_UI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Captured files</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QTreeWidget" name="treeWidget" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>File name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Size</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="deleteButton" >
     <property name="text" >
      <string>Delete current file</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="toggleButton" >
     <property name="text" >
      <string>Toggle selection</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManageCaptures_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManageCaptures_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
